scripts/preprocessing/04b-run_ssp.py

```diff
@@ -79,10 +83,15 @@ def run_ssp(*, cfg, subject, session=None):
     else:
         ch_names = None
 
-    reject_eog_ = config.get_ssp_reject(
-        ssp_type='eog',
-        epochs=(create_eog_epochs(raw)
-                if cfg.ssp_reject_eog == 'autoreject_global' else None))
+    reject_eog_ = None
+    eog_epochs = None
+    if cfg.ssp_reject_eog == 'autoreject_global':
+        eog_epochs = create_eog_epochs(raw)
+        if len(eog_epochs) >= 5:
```

Comment:
How about we introduce an option here? `MIN_SSP_N_EOG` or something similar? Perhaps we want allow larger numbers.

Reply:
We don't need to restrict this to SSP -> how about `min_eog_epochs : None | float`, and it can be used for SSP and ICA alike?